cmd/statistics/main.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2016 ISRG.  All rights reserved
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package main
+
+import (
+	"os"
+
+	"github.com/codegangsta/cli"
+	"github.com/jmhodges/clock"
+
+	"github.com/letsencrypt/boulder/cmd"
+	blog "github.com/letsencrypt/boulder/log"
+	"github.com/letsencrypt/boulder/metrics"
+	"github.com/letsencrypt/boulder/sa"
+	"github.com/letsencrypt/boulder/statistics"
+)
+
+func main() {
+	app := cmd.NewAppShell("statistics", "Generates statistics about Boulder")
```

Comment:
We are also trying to move towards using just `flag`, since you aren't using many args the switch here should be pretty simple.